assignments/objects.js

```diff
@@ -19,26 +12,73 @@ const example = {
 
 // Write your intern objects here:
 
+class Intern {
+  constructor(id, name, email, gender){
+  this.id = id;
+  this.name = name;
+  this.email = email;
+  this.gender = gender;
+  }
+}
+
+// 1, mmelloy0@psu.edu, Mitzi, F
+let mitzi = new Intern(1, "Mitzi", "mmelloy0@psu.edu", "Female" );
+
+// 2,kdiben1@tinypic.com,Kennan, M
+let kennan = new Intern(2, "Kennan", "kdiben1@tinypic.com", "Male");
+
+// 3,kmummery2@wikimedia.org,Keven, M
+let keven = new Intern(3, "Keven", "kmummery2@wikimedia.org", "Male");
+
+// 4,gmartinson3@illinois.edu,Gannie, M
+let gannie = new Intern(4, "Gannie", "gmartinson3@illinois.edu", "Male");
+
+// 5,adaine5@samsung.com,Antonietta, F
+let antonietta = new Intern(5, "Antonietta", "adaine5@samsung.com", "Female");
+
+
```

Comment:
Nice job here all you had to do was make them all objects but you went above and beyond and made a class awesome